l10n/templates/strings.properties

```diff
@@ -131,3 +138,4 @@ pocket_feedback_header=The best of the web, curated by over 25 million people.
 # (pocket_feedback_header) to provide more information about Pocket.
 pocket_feedback_body=Pocket, a part of the Mozilla family, will help connect you to high-quality content that you may not have found otherwise.
 pocket_send_feedback=Send Feedback
+empty_state_topstories=You’ve caught up. Check back later for more top stories from Pocket. Can’t wait? Select a popular topic to find more great stories from around the web.
```

Comment:
If that’s the case, can we use `{provider}` here and fix the code later? At least we expose directly the right string.

In that’s the case, it needs a localization note explaining what `{provider}` is.